src/main/kotlin/pl/edu/uj/ii/ksi/mordor/persistence/entities/FileContent.kt

```diff
@@ -12,7 +13,7 @@ data class FileContent(
     var id: Long? = null,
 
     @MapsId
-    @OneToOne(optional = false)
+    @OneToOne(optional = false, cascade = [CascadeType.ALL], orphanRemoval = true)
```

Comment:
I'm not sure on what side of the relation orphanRemoval should be set, please double check.

Reply:
orphanRemoval should be placed in parent, so this should be removed